lib/internal/modules/cjs/loader.js

```diff
@@ -1272,9 +1248,9 @@ Module._extensions['.js'] = function(module, filename) {
     content = fs.readFileSync(filename, 'utf8');
   }
   if (StringPrototypeEndsWith(filename, '.js')) {
-    const pkg = readPackageScope(filename);
+    const pkg = readPackageScope(filename) || {};
```

Comment:
```suggestion
    const pkg = readPackageScope(filename) || { __proto__: null };
```

You potentially read from this fall back below, which is susceptible to prototype pollution.